source/slang/hlsl.meta.slang

```diff
@@ -9725,7 +9724,8 @@ void TraceMotionRay(
         let tmax = Ray.TMax;
 
         p = Payload;
-        spirv_asm {
+        spirv_asm 
```

Comment:
This function and others in hlsl.meta.slang, such as TraceRay and TraceMotionRay, needs [require(spirv, raytracing)]